cmake/configs/x86/windows/config.cmake

```diff
@@ -8,8 +8,6 @@ set(ENABLE_STATIC ON CACHE BOOL "" FORCE)
 set(ENABLE_SHARED OFF CACHE BOOL "" FORCE)
 set(WITH_PIC ON CACHE BOOL "" FORCE)
 
-# Compile as C code .
```

Comment:
Do you know why this was originally added? It seems odd to just treat everything on Windows as C. I'm a little worried that removing it will cause some C to get treated as C++, but if the CI passes it's probably OK.

Reply:
I couldn't find anything in git history. It is odd and cmake was doing the correct identification of C vs C++ files and setting the appropriate flags for VS. If it passes CI then its good for me.